app/components/moderation-list-row/component.js

```diff
@@ -39,35 +39,56 @@ export default Component.extend({
 
     noActions: computed.not('submission.actions.length'),
 
+    // latest action attempted on the preprint
     latestAction: computed('submission.actions.[]', function() {
         return latestAction(this.get('submission.actions'));
     }),
 
+    // first three contributors to the preprint
     firstContributors: computed('submission.node.contributors', function() {
         return this.get('submission.node.contributors').slice(0, 3);
     }),
 
+    // count of contributors in-addition to the first three
     additionalContributors: computed('submission.node.contributors', function() {
         return this.get('submission.node.contributors.content.meta.pagination.total') - 3;
     }),
 
+    // check if it has been more than a day since preprint last transition
     gtDay: computed('submission.dateLastTransitioned', function() {
         return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
     }),
 
-    relevantDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
+    // date of preprint acceptance or rejection
+    acceptedRejectedtDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
         return this.get('gtDay') ?
             moment(this.get('submission.dateLastTransitioned')).format('MMMM DD, YYYY') :
             moment(Math.min(Date.parse(this.get('submission.dateLastTransitioned')), Date.now())).fromNow();
     }),
 
-    // translations
-    statusTimeDate: computed('submission.reviewsState', 'gtDay', 'noActions', function() {
+    // date of preprint submission
+    submitDate: computed('submission.dateCreated', 'gtDay', function() {
+        return this.get('gtDay') ?
```

Comment:
Using `gtDay` here seems wrong; it looks at `submission.dateLastTransitioned`, not `submission.dateCreated`. So if something was submitted more than a day ago, but accepted less than a day ago, this will format the date wrong.

Now that we have two dates displayed, it might make sense to remove the `gtDay` property and just do that calculation in each place we need it (assuming that can be done without duplicated code).